java/test/jmri/InstanceManagerTest.java

```diff
@@ -132,6 +137,22 @@ public void testGenericStoreAndGetTwoDifferentTypes() {
 
         Assert.assertEquals("retrieved same PowerManager", m1, m2);
         Assert.assertEquals("retrieved same TurnoutManager", t1, t2);
+
+        Assert.assertEquals("access by string",
+                InstanceManager.getDefault(PowerManager.class),
+                InstanceManager.getDefault("jmri.PowerManager")
+            );
+
+    }
+
+    @Test
+    public void testGetInstance() throws ClassNotFoundException {
+        // first check a predicate - Class.forName returns same object always
+        Assert.assertTrue("access by string",
+                Class.forName("jmri.PowerManager") == Class.forName("jmri.PowerManager")
+            );
```

Comment:
Just checking. Is this test method complete? I ask since the comment gives the sense that there is more to come. And the method name implies something else.